### ISSUE TITLE
Remove JITSERVER_ENABLE_SSL once jitaas is merged into master

### ISSUE THREAD

**Issue by a7ehuo** [eclipse/openj9#6327]
`JITSERVER_ENABLE_SSL` is added to conditionally build JITaaS with OpenSSL. This flag should be removed after `jitaas` branch is merged into `master` branch. JITaaS + OpenSSL should be properly tested again when OpenSSL is enabled by default.
